packages/core/src/Engine.ts

```diff
@@ -242,11 +247,16 @@ export class Engine extends EventDispatcher {
     const scene = this._sceneManager._activeScene;
     const componentsManager = this._componentsManager;
     if (scene) {
+      const cameras = scene._activeCameras;
+      /** Sort on priority. */
+      cameras.length > 0 && cameras.sort((camera1, camera2) => camera1.priority - camera2.priority);
+
       componentsManager.callScriptOnStart();
       componentsManager.callScriptOnUpdate(deltaTime);
       componentsManager.callAnimationUpdate(deltaTime);
       componentsManager.callScriptOnLateUpdate(deltaTime);
 
+      this._inputManager._update();
```

Comment:
inputManager should  update before `componentsManager.callScriptOnUpdate(deltaTime);`